app/src/test/java/com/onboarding/parkingsystemjava/mvp/presenter/ConfigureParkingPresenterTest.java

```diff
@@ -0,0 +1,47 @@
+package com.onboarding.parkingsystemjava.mvp.presenter;
+
+import com.onboarding.parkingsystemjava.listener.OnInputListener;
+import com.onboarding.parkingsystemjava.mvp.contract.ConfigureParkingLotsContract;
+import com.onboarding.parkingsystemjava.utils.ConstUtils;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class ConfigureParkingPresenterTest {
+    ConfigureParkingLotsContract.ConfigureParkingPresenter presenter;
+    @Mock
+    ConfigureParkingLotsContract.ConfigureParkingView view;
+    @Mock
+    OnInputListener listener;
+
+    @Before
+    public void setUp() {
+        presenter = new ConfigureParkingPresenter(listener, view);
+    }
+
+    @Test
+    public void onCancelButtonPressTest() {
+        presenter.OnCancelButtonPress();
+        verify(view).closeDialog();
+    }
+
+    @Test
+    public void onOkButtonPressTestPass() {
+        Mockito.when(view.getLots()).thenReturn(ConstUtils.PARKING_LOT_STRING);
```

Comment:
you should not use the constants from the ConstUtils here, please use const local variables